### ISSUE TITLE
Request: formatting exisiting links

### ISSUE THREAD

**Issue by santiyounger** [denolehov/obsidian-url-into-selection#2]
this plugin is great, I find it extremely useful.

Currently the url is pasted into existing text

Would it be possible to do the reverse?

adding text to an existing url

this is due to my current situation where I have some unformulated urls in my notes

So if for instance if I have the URL https://github.com/denolehov/obsidian-url-into-selection

in obsidian there's the shortcut to format urls, 
through `ctrl`+`k` would format it like this 
```
[https://github.com/denolehov/obsidian-url-into-selection]()
```
what's I'd need is the opposite, a shortcut to 

format things like this
```
[](https://github.com/denolehov/obsidian-url-into-selection)
```

I know this might be out of the scope of your plug in, but I'd love to know if your existing code would allow for such an additional option

Thanks for the great plugin again!

**Comment by denolehov** [comment]
Hey :) This is a very interesting request! I have implemented it in `v1.1.0`, feel free to update the plugin! 

Let me know if you have any questions! 

:heart:

**Comment by santiyounger** [comment]
amazing @denolehov  wow thank you for adding it, it's exactly what I needed, really appreciate it!
Keep up the awesome work!
